sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/http/DefaultHttpClient.java

```diff
@@ -158,7 +159,7 @@ private static String doHttpCall(String urlString, String method, Map<String, St
             MobileCenterLog.verbose(LOG_TAG, "HTTP response status=" + status + " payload=" + response);
 
             /* Generate exception on failure. */
-            if (status != 200)
+            if (status != HttpURLConnection.HTTP_OK)
```

Comment:
Should we use `status >= 200 && status < 300` here?

Reply:
Good point. We should change that.